<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLogin</class>
 <widget class="QDialog" name="UserLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#UserLogin{
	border-image: url(:/new/prefix1/user.png);
	border-image: url(:/new/prefix1/back123.jpg);

}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #0099ff, stop:1 #00ff99);
	background-color: rgb(26, 144, 148);
	background-color:transparent;
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>290</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Fax</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_3{
color: rgb(255, 255, 255);
border: 1px solid rgb(18, 31, 63);
border-radius: 19px;
font-weight:bold;
background-color:transparent;
padding-left:78px;
}

#lineEdit_3:hover{
background-color: rgb(16, 23, 49);
}

#lineEdit_3:focus{
border-radius:20px;
border: 1.2px solid rgb(26, 42, 74) ;
background-color:rgb(44, 82, 153);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>290</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Fax</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_2{
color: rgb(255, 255, 255);
border: 1px solid rgb(18, 31, 63);
border-radius: 19px;
font-weight:bold;
background-color:transparent;
padding-left:78px;
}

#lineEdit_2:hover{
background-color: rgb(16, 23, 49);
}

#lineEdit_2:focus{
border-radius:20px;
border: 1.2px solid rgb(26, 42, 74) ;
background-color:rgb(44, 82, 153);
}</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
background-color:white;
color: rgb(15, 31, 61);
boder: 1.2px solid rgb(80, 100, 187);
border-radius: 20px;
font: 63 11pt &quot;Lucida Bright&quot;;
font-weight:bold;
font-size: 15px;
letter-spacing: 5px
}

#pushButton:hover{
border: 1px solid rgb(67, 200, 200);
background-color: green;
color:white;
}</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Bright</family>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background:transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>User's login</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>770</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
